.github/workflows/ci.yml

```diff
@@ -130,7 +130,7 @@ jobs:
       - name: Install system dependencies
         run: |
           sudo apt update
-          sudo apt install gir1.2-appindicator3-0.1 libgirepository1.0-dev python3-tk
+          sudo apt install gir1.2-appindicator3-0.1 libgirepository1.0-dev gir1.2-ayatanaappindicator3-0.1 libayatana-appindicator3-1 python3-tk
```

Comment:
If the intention is to move away from the regular AppIndicator implementation to Ayatana's, you can drop the `gir1.2-appindicator3-0.1` package here.